lib/wallet.js

```diff
@@ -15,14 +16,22 @@ const _clientFund = new WeakMap();
  * A class for performing various operations on a wallet.
  * @alias module:nahmii-sdk
  */
-class Wallet extends ethers.Wallet {
+class Wallet extends ethers.Signer {
```

Comment:
What is the rationale behind inheriting directly from "Signer" rather than the "Wallet" that provides an implmentation of it as well?

Reply:
Wallet comes with a lot of functionality which is only relevant to software wallets. The recommended way to add different ways of signing according to the docs is to extend the Signer class.